src/Icon/Icon.js

```diff
@@ -27,9 +28,9 @@ export default class Icon extends React.Component {
     };
   }
 
-  template({ icon, className, style, onClick, paths }) {
+  template({ icon, color, className, style = {}, onClick, paths }) {
     return icon ? (
-      <i className={cx('icon', `icon-${icon}`, className)} style={style} onClick={onClick}>
+      <i className={cx('icon', `icon-${icon}`, className)} style={{ ...style, color: color || style.color }} onClick={onClick}>
```

Comment:
if you write it like this:
```jsx
style={{ ...style, color }}
```
does `color` overwrite `style.color` even when `color` is `undefined`?